timelock-impl/src/main/java/com/palantir/atlasdb/timelock/lock/BlockingTimeLimitedLockService.java

```diff
@@ -212,7 +212,7 @@ private BlockingTimeoutException logAndHandleTimeout(LockRequestSpecification sp
         log.info(logMessage,
                 SafeArg.of("timeoutDurationMillis", blockingTimeLimitMillis),
                 SafeArg.of("method", specification.method()),
-                SafeArg.of("client", specification.client()),
+                UnsafeArg.of("client", specification.client()),
```

Comment:
Why do we consider clients to be Unsafe?  Can discuss internally but this is extremely useful information and it isn't sensitive IMO.

Reply:
I think this one is safe to discuss openly, actually. The main thing is that this is "user" (as in Atlas-client) provided. I'd be okay with making/leaving them as Safe, but it does mean that an Atlas client accidentally making lock requests with unsafe clients would mean that this is unsafe.

Reply:
Actually, let's discuss offline. I checked where this came from and it should be safe